### DISCUSSION TITLE
Return of v2 removed features?

### DISCUSSION THREAD

**Discussion by decentral1se** [dexidp/dex#3423] [General]
https://dexidp.io/docs/v2/#removed-features

> ... For the most part we aim to add most of these features back into v2 ...

Did they return? I understand there is no web UI registration flow supported in v2 still.

**Comment by nabokihms** [comment]
Probably they are gone forever. Which feature do you want to return?

**Reply by decentral1se** [reply]
Oh, is this then a docs issue? 🙃 Well all of them are what I need and I'm curious what people are doing? Otherwise did people go back to Keycloak/Authentik for all the user management / email / invite / etc. functionality?

**Reply by nabokihms** [reply]
I would say yes because there are already many great providers with features that Dex will never be able to match.

Instead, for Dex as a project, we are focusing on:
- security
- stability
- integration with other products
- giving standard OIDC interface

User management in Dex only exists for testing/bootstrapping purposes or for some small projects/dev environments.
